support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -21,15 +21,15 @@ object DigitalPackEmailFields {
           paymentFields(pm, directDebitMandateId) ++
             directReaderFields(promotion, billingPeriod, user, currency, paymentSchedule)
         case None /*Corporate*/ => List(
-          "Subscription details" -> "Group subscription"
+          "subscription_details" -> "Group subscription"
         )
       }
 
     val fields = List(
-      "ZuoraSubscriberId" -> subscriptionNumber,
-      "EmailAddress" -> user.primaryEmailAddress,
-      "First Name" -> user.firstName,
-      "Last Name" -> user.lastName,
+      "zuorasubscriberid" -> subscriptionNumber,
```

Comment:
I can see that this key used to be CamelCase and so results in no underscores after the transformation. Just curious if there is any particular reason the keys followed different conventions to start with (e.g. `ZuoraSubscriberId` vs `First Name`.

Reply:
not sure really, it's possible some of them are unused now, but it's hard to tell without going through braze.
They would have been added at different times by different people though and as long as they match at our end and braze end, there's no additional requirement on them.